clearwater-sip-stress-coreonly.root/usr/share/clearwater/bin/run_stress

```diff
@@ -52,15 +54,17 @@ class SIPpstats(csv.excel):
 SIPP_BIN = "/usr/share/clearwater/bin/sipp_coreonly"
 SCRIPTS_DIR = "/usr/share/clearwater/sip-stress/"
 
-REGISTER_CSV = "/tmp/registers.csv"
-INVITE_CSV = "/tmp/calls.csv"
+REGISTER_CSV = NamedTemporaryFile(prefix="sipp_register")
+INVITE_CSV = NamedTemporaryFile(prefix="sipp_calls")
```

Comment:
You should probably pass in `delete=False` here. The files are dangerously close to getting deleted automatically (in particular, if someone ever changes the script to close the file handles with `.close()`, the files will suddenly vanish).